float32[] rightGripperPosition
float32[] rightGripperOrientation
float32[] leftGripperPosition
float32[] leftGripperOrientation